2024-11-17 22:38:17,676 - INFO - Iniciando testes para ISBN: 9781449355739
2024-11-17 22:38:21,287 - INFO - Total de APIs testadas: 6
2024-11-17 22:38:21,295 - INFO - APIs bem-sucedidas: 1
2024-11-17 22:38:21,297 - INFO - Relatórios gerados com sucesso:
2024-11-17 22:38:21,298 - INFO - HTML: reports/isbn_report_9781449355739_20241117_223821.html
2024-11-17 22:38:21,298 - INFO - JSON: reports/isbn_report_9781449355739_20241117_223821.json
2024-11-17 22:38:21,298 - INFO - Iniciando testes para ISBN: 9781617294136
2024-11-17 22:38:21,531 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /en/jfkdsj
2024-11-17 22:38:22,406 - INFO - Total de APIs testadas: 6
2024-11-17 22:38:22,407 - INFO - APIs bem-sucedidas: 2
2024-11-17 22:38:22,409 - INFO - Relatórios gerados com sucesso:
2024-11-17 22:38:22,409 - INFO - HTML: reports/isbn_report_9781617294136_20241117_223822.html
2024-11-17 22:38:22,410 - INFO - JSON: reports/isbn_report_9781617294136_20241117_223822.json
2024-11-17 22:38:22,410 - INFO - Iniciando testes para ISBN: 9780134685991
2024-11-17 22:38:22,620 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /en/jfkdsj
2024-11-17 22:38:23,576 - INFO - Total de APIs testadas: 6
2024-11-17 22:38:23,577 - INFO - APIs bem-sucedidas: 1
2024-11-17 22:38:23,579 - INFO - Relatórios gerados com sucesso:
2024-11-17 22:38:23,579 - INFO - HTML: reports/isbn_report_9780134685991_20241117_223823.html
2024-11-17 22:38:23,579 - INFO - JSON: reports/isbn_report_9780134685991_20241117_223823.json
2024-11-18 00:22:00,747 - INFO - Iniciando testes para ISBN: 9781449355739
2024-11-18 00:22:06,067 - INFO - Total de APIs testadas: 6
2024-11-18 00:22:06,068 - INFO - APIs bem-sucedidas: 1
2024-11-18 00:22:06,071 - INFO - Relatórios gerados com sucesso:
2024-11-18 00:22:06,071 - INFO - HTML: reports/isbn_report_9781449355739_20241118_002206.html
2024-11-18 00:22:06,071 - INFO - JSON: reports/isbn_report_9781449355739_20241118_002206.json
2024-11-18 00:22:06,071 - INFO - Iniciando testes para ISBN: 9781617294136
2024-11-18 00:22:06,287 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /en/jfkdsj
2024-11-18 00:22:08,254 - INFO - Total de APIs testadas: 6
2024-11-18 00:22:08,255 - INFO - APIs bem-sucedidas: 2
2024-11-18 00:22:08,258 - INFO - Relatórios gerados com sucesso:
2024-11-18 00:22:08,258 - INFO - HTML: reports/isbn_report_9781617294136_20241118_002208.html
2024-11-18 00:22:08,258 - INFO - JSON: reports/isbn_report_9781617294136_20241118_002208.json
2024-11-18 00:22:08,259 - INFO - Iniciando testes para ISBN: 9780134685991
2024-11-18 00:22:08,468 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /en/jfkdsj
2024-11-18 00:22:10,462 - INFO - Total de APIs testadas: 6
2024-11-18 00:22:10,462 - INFO - APIs bem-sucedidas: 1
2024-11-18 00:22:10,464 - INFO - Relatórios gerados com sucesso:
2024-11-18 00:22:10,464 - INFO - HTML: reports/isbn_report_9780134685991_20241118_002210.html
2024-11-18 00:22:10,465 - INFO - JSON: reports/isbn_report_9780134685991_20241118_002210.json
2024-11-18 00:33:31,279 - ERROR - Erro processando ISBN 9781449355739: module 'isbnlib' has no attribute 'ext'
2024-11-18 00:33:31,280 - ERROR - Erro processando ISBN 9781617294136: module 'isbnlib' has no attribute 'ext'
2024-11-18 00:33:31,280 - ERROR - Erro processando ISBN 9780134685991: module 'isbnlib' has no attribute 'ext'
2024-11-18 00:38:03,768 - ERROR - Erro processando ISBN 9781449355739: module 'isbnlib' has no attribute 'registry'
2024-11-18 00:38:03,768 - ERROR - Erro processando ISBN 9781617294136: module 'isbnlib' has no attribute 'registry'
2024-11-18 00:38:03,769 - ERROR - Erro processando ISBN 9780134685991: module 'isbnlib' has no attribute 'registry'
2024-11-18 00:39:54,087 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781449355739&maxRecs=1 with code 404 [Not Found]
2024-11-18 00:39:56,307 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781449355739 with code 403 [Forbidden]
2024-11-18 00:39:58,579 - ERROR - Some error on editions 'merge' service for 9781449355739!
2024-11-18 00:40:00,985 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781449355739 with code 400 [Bad Request]
2024-11-18 00:40:04,535 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781617294136&maxRecs=1 with code 404 [Not Found]
2024-11-18 00:40:05,497 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781617294136 with code 403 [Forbidden]
2024-11-18 00:40:06,776 - ERROR - Some error on editions 'merge' service for 9781617294136!
2024-11-18 00:40:08,130 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781617294136 with code 400 [Bad Request]
2024-11-18 00:40:13,649 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9780134685991&maxRecs=1 with code 404 [Not Found]
2024-11-18 00:40:15,940 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9780134685991 with code 403 [Forbidden]
2024-11-18 00:40:17,527 - ERROR - Some error on editions 'merge' service for 9780134685991!
2024-11-18 00:40:18,828 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9780134685991 with code 400 [Bad Request]
2024-11-18 00:48:36,566 - ERROR - Erro processando ISBN 9781449355739: __main__.APITestResult() got multiple values for keyword argument 'cache_hit'
2024-11-18 00:48:36,566 - ERROR - Erro processando ISBN 9781617294136: __main__.APITestResult() got multiple values for keyword argument 'cache_hit'
2024-11-18 00:48:36,566 - ERROR - Erro processando ISBN 9780134685991: __main__.APITestResult() got multiple values for keyword argument 'cache_hit'
2024-11-19 23:10:48,875 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781449355739&maxRecs=1 with code 404 [Not Found]
2024-11-19 23:10:49,948 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781449355739 with code 403 [Forbidden]
2024-11-19 23:10:51,373 - ERROR - Some error on editions 'merge' service for 9781449355739!
2024-11-19 23:10:53,752 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781449355739 with code 400 [Bad Request]
2024-11-19 23:10:57,993 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781617294136&maxRecs=1 with code 404 [Not Found]
2024-11-19 23:10:58,967 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781617294136 with code 403 [Forbidden]
2024-11-19 23:11:00,376 - ERROR - Some error on editions 'merge' service for 9781617294136!
2024-11-19 23:11:02,042 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781617294136 with code 400 [Bad Request]
2024-11-19 23:11:09,171 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9780134685991&maxRecs=1 with code 404 [Not Found]
2024-11-19 23:11:10,356 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9780134685991 with code 403 [Forbidden]
2024-11-19 23:11:11,849 - ERROR - Some error on editions 'merge' service for 9780134685991!
2024-11-19 23:11:13,423 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9780134685991 with code 400 [Bad Request]
2024-11-20 12:20:51,291 - ERROR - Erro processando ISBN 9786586057485: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9788575228289: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9788550800653: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9781718502666: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9781593279288: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9781492056355: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,292 - ERROR - Erro processando ISBN 9781492051367: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,293 - ERROR - Erro processando ISBN 9781119775553: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:20:51,293 - ERROR - Erro processando ISBN 9781118951309: 'ISBNAPITester' object has no attribute 'test_deutsche_nb'
2024-11-20 12:25:32,001 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9786586057485&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:25:35,474 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9786586057485 with code 403 [Forbidden]
2024-11-20 12:25:38,537 - ERROR - Some error on editions 'merge' service for 9786586057485!
2024-11-20 12:25:40,903 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9786586057485 with code 400 [Bad Request]
2024-11-20 12:25:43,024 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce1850>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9786586057485
2024-11-20 12:25:44,032 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce1c40>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9786586057485
2024-11-20 12:25:44,185 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce1ee0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9786586057485
2024-11-20 12:25:45,194 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf4340>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9786586057485
2024-11-20 12:25:46,041 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce12b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9786586057485
2024-11-20 12:25:47,203 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf47f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9786586057485
2024-11-20 12:25:53,371 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9788575228289&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:25:54,657 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9788575228289 with code 403 [Forbidden]
2024-11-20 12:25:55,742 - ERROR - Some error on editions 'merge' service for 9788575228289!
2024-11-20 12:25:57,275 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9788575228289 with code 404 [Not Found]
2024-11-20 12:25:57,292 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a91a30>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788575228289
2024-11-20 12:25:57,720 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf4a90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788575228289
2024-11-20 12:25:58,298 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a91580>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788575228289
2024-11-20 12:25:58,725 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4400>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788575228289
2024-11-20 12:26:00,307 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00670>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788575228289
2024-11-20 12:26:00,733 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce1ac0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788575228289
2024-11-20 12:26:23,733 - CRITICAL - ISBNLibURLError for http://classify.oclc.org/classify2/Classify?isbn=9788550800653&maxRecs=1 with reason timed out
2024-11-20 12:26:33,748 - CRITICAL - ISBNLibURLError for https://openlibrary.org/search.json?q=9788550800653&fields=isbn with reason timed out
2024-11-20 12:26:33,828 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9788550800653 with code 403 [Forbidden]
2024-11-20 12:26:34,762 - ERROR - Some error on editions 'merge' service for 9788550800653!
2024-11-20 12:26:36,094 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9788550800653 with code 400 [Bad Request]
2024-11-20 12:26:36,107 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105c8e820>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788550800653
2024-11-20 12:26:36,599 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce11f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788550800653
2024-11-20 12:26:37,115 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf4400>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788550800653
2024-11-20 12:26:37,606 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d006a0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788550800653
2024-11-20 12:26:39,122 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00130>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9788550800653
2024-11-20 12:26:39,615 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d003d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9788550800653
2024-11-20 12:26:46,172 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.oclc.org', port=443): Read timed out. (read timeout=10)")': /en/jfkdsj
2024-11-20 12:26:48,947 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781718502666&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:26:50,130 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781718502666 with code 403 [Forbidden]
2024-11-20 12:26:51,898 - ERROR - Some error on editions 'merge' service for 9781718502666!
2024-11-20 12:26:53,597 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781718502666 with code 400 [Bad Request]
2024-11-20 12:26:53,611 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a9aac0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781718502666
2024-11-20 12:26:54,356 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105ce1580>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781718502666
2024-11-20 12:26:54,618 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a9a070>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781718502666
2024-11-20 12:26:55,363 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4190>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781718502666
2024-11-20 12:26:56,627 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00c10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781718502666
2024-11-20 12:26:57,370 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00760>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781718502666
2024-11-20 12:26:57,614 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 12:26:59,562 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781593279288&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:27:00,543 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781593279288 with code 403 [Forbidden]
2024-11-20 12:27:01,892 - ERROR - Some error on editions 'merge' service for 9781593279288!
2024-11-20 12:27:03,834 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781593279288 with code 400 [Bad Request]
2024-11-20 12:27:03,848 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00dc0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781593279288
2024-11-20 12:27:04,855 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa46a0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781593279288
2024-11-20 12:27:05,551 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf4af0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781593279288
2024-11-20 12:27:06,558 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781593279288
2024-11-20 12:27:06,862 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105cf4a90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781593279288
2024-11-20 12:27:08,565 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4880>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781593279288
2024-11-20 12:27:13,849 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='openlibrary.org', port=443): Read timed out. (read timeout=10)")': /api/books?bibkeys=ISBN%3A9781593279288&format=json&jscmd=data
2024-11-20 12:27:13,849 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPConnectionPool(host='classify.oclc.org', port=80): Read timed out. (read timeout=10)")': /classify2/Classify?isbn=9781593279288&summary=true
2024-11-20 12:27:13,850 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x105ce1280>, 'Connection to api.crossref.org timed out. (connect timeout=10)')': /works?query.bibliographic=9781593279288&filter=type%3Abook&select=title%2Cauthor%2Cpublished-print%2Cpublisher
2024-11-20 12:27:16,872 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='www.mybib.com', port=443): Read timed out. (read timeout=10)")': /api/v1/references?q=9781593279288
2024-11-20 12:27:18,865 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='api.springernature.com', port=443): Read timed out. (read timeout=10)")': /metadata/book/isbn/9781593279288
2024-11-20 12:27:24,857 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x105d2b400>, 'Connection to openlibrary.org timed out. (connect timeout=10)')': /api/books?bibkeys=ISBN%3A9781593279288&format=json&jscmd=data
2024-11-20 12:27:24,858 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x105d2b970>, 'Connection to api.crossref.org timed out. (connect timeout=10)')': /works?query.bibliographic=9781593279288&filter=type%3Abook&select=title%2Cauthor%2Cpublished-print%2Cpublisher
2024-11-20 12:27:29,874 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x105d2bf40>, 'Connection to api.springernature.com timed out. (connect timeout=10)')': /metadata/book/isbn/9781593279288
2024-11-20 12:27:32,034 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 12:27:34,581 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781492056355&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:27:35,563 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781492056355 with code 403 [Forbidden]
2024-11-20 12:27:36,769 - ERROR - Some error on editions 'merge' service for 9781492056355!
2024-11-20 12:27:37,876 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781492056355 with code 400 [Bad Request]
2024-11-20 12:27:37,895 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2b820>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492056355
2024-11-20 12:27:38,359 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2b7f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492056355
2024-11-20 12:27:38,901 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2bf70>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492056355
2024-11-20 12:27:39,364 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa42b0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492056355
2024-11-20 12:27:40,907 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2b280>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492056355
2024-11-20 12:27:41,370 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4a190>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492056355
2024-11-20 12:27:41,401 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 12:27:45,627 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781492051367&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:27:46,877 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781492051367 with code 403 [Forbidden]
2024-11-20 12:27:48,802 - ERROR - Some error on editions 'merge' service for 9781492051367!
2024-11-20 12:27:50,221 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781492051367 with code 400 [Bad Request]
2024-11-20 12:27:50,248 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2b910>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492051367
2024-11-20 12:27:50,684 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4a070>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492051367
2024-11-20 12:27:51,255 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4af70>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492051367
2024-11-20 12:27:51,691 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105c83490>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492051367
2024-11-20 12:27:53,264 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4430>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781492051367
2024-11-20 12:27:53,697 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4afd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781492051367
2024-11-20 12:27:53,728 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 12:27:56,996 - CRITICAL - ISBNLibHTTPError for http://classify.oclc.org/classify2/Classify?isbn=9781119775553&maxRecs=1 with code 404 [Not Found]
2024-11-20 12:27:58,199 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781119775553 with code 403 [Forbidden]
2024-11-20 12:27:59,645 - CRITICAL - ISBNLibHTTPError for https://en.wikipedia.org/api/rest_v1/data/citation/mediawiki/9781119775553 with code 404 [Not Found]
2024-11-20 12:27:59,647 - ERROR - Some error on editions 'merge' service for 9781119775553!
2024-11-20 12:28:00,813 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781119775553 with code 400 [Bad Request]
2024-11-20 12:28:00,827 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4a580>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781119775553
2024-11-20 12:28:01,273 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103aa4c10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781119775553
2024-11-20 12:28:01,834 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a918e0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781119775553
2024-11-20 12:28:02,279 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d4a3d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781119775553
2024-11-20 12:28:03,840 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2f9a0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781119775553
2024-11-20 12:28:04,287 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2bac0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781119775553
2024-11-20 12:28:26,824 - CRITICAL - ISBNLibURLError for http://classify.oclc.org/classify2/Classify?isbn=9781118951309&maxRecs=1 with reason timed out
2024-11-20 12:28:30,113 - CRITICAL - ISBNLibHTTPError for http://www.librarything.com/api/thingISBN/9781118951309 with code 403 [Forbidden]
2024-11-20 12:28:31,387 - CRITICAL - ISBNLibHTTPError for https://en.wikipedia.org/api/rest_v1/data/citation/mediawiki/9781118951309 with code 404 [Not Found]
2024-11-20 12:28:31,391 - ERROR - Some error on editions 'merge' service for 9781118951309!
2024-11-20 12:28:33,027 - CRITICAL - ISBNLibHTTPError for http://dx.doi.org/9781118951309 with code 400 [Bad Request]
2024-11-20 12:28:33,041 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d00910>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781118951309
2024-11-20 12:28:33,516 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a9ae50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781118951309
2024-11-20 12:28:34,048 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x103a9a640>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781118951309
2024-11-20 12:28:34,523 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2f8e0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781118951309
2024-11-20 12:28:36,055 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2f760>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/isbn/9781118951309
2024-11-20 12:28:36,531 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x105d2f130>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /v1/search?q=isbn%3A9781118951309
2024-11-20 12:28:36,566 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 13:12:27,145 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 13:12:27,145 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 13:12:27,145 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 13:12:27,145 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 13:12:27,147 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:04:35,804 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:04:35,805 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:04:35,805 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:04:35,805 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:04:35,807 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:06:09,670 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:06:09,671 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:06:09,671 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:06:09,671 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:06:09,673 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:09:47,141 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:09:47,141 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:09:47,141 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:09:47,142 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:09:47,143 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:12:40,263 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:12:40,263 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:12:40,263 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:12:40,264 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:12:40,265 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:32:33,288 - ERROR - Erro processando ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:32:33,289 - ERROR - Erro processando ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:32:33,289 - ERROR - Erro processando ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:32:33,289 - ERROR - Erro processando ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:32:33,291 - ERROR - Erro processando ISBN 9781118951309: unhashable type: 'list'
2024-11-20 16:34:54,720 - ERROR - Erro processando resultados para ISBN 9781718502666: unhashable type: 'list'
2024-11-20 16:34:54,721 - ERROR - Erro processando resultados para ISBN 9781593279288: unhashable type: 'list'
2024-11-20 16:34:54,721 - ERROR - Erro processando resultados para ISBN 9781492056355: unhashable type: 'list'
2024-11-20 16:34:54,721 - ERROR - Erro processando resultados para ISBN 9781492051367: unhashable type: 'list'
2024-11-20 16:34:54,722 - ERROR - Erro processando resultados para ISBN 9781118951309: unhashable type: 'list'
